src/entities/user.ts

```diff
@@ -104,6 +104,18 @@ export class User {
     })
     confirmationToken: string | null
 
+    @Column({
+        type: 'varchar',
+        nullable: true
+    })
+    resetToken: string | null
```

Comment:
can you add `select: false` for token (email and password) and timestamp. See the password field.

Reply:
Ok, but you will have to explain why ;)